src/components/SentenceReport/SentenceReportContent.tsx

```diff
@@ -55,10 +55,6 @@ const SentenceReportContent = ({ context }) => {
                 </Col>
             )}
             <Col span={24}>
-                <Paragraph className="title">
```

Comment:
question: I don't see an issue so I didn't understand the reason of this to be removed, can you elaborate?

Reply:
The problem was, when I changed the SourcesList component I added the tittle on the component and removed from ClaimViewPage, but this component was being used on SentenceReportContent causing the duplication of the titles: 
![image](https://github.com/AletheiaFact/aletheia/assets/99989478/b8a41d6e-55e4-486e-b43b-944fc74da73e)

ps: But thinking again I gonna change that cause I think its better for the tittle to be outside of the component. What do you think ?

Reply:
Agreed.